src/main/scala/io/scalac/tezos/translator/routes/dto/LibraryEntryDTO.scala

```diff
@@ -21,7 +25,7 @@ object LibraryEntryDTO {
 }
 
 case class LibraryEntryRoutesAdminDto(
-                                       uid: String,
+                                       uid: LibraryEntryId,
                                        title: String,
```

Comment:
Why not to introduce refined types for `title`, `author` and so on ? (Just asking)

Reply:
It's was my plan for the second pr